test/integration/features/#basic.feature#

```diff
@@ -0,0 +1,100 @@
+Feature: Basic test
+    Checks whether CRC top-level commands behave correctly.
+
+    @darwin @linux @windows
+    Scenario: CRC version
+        When executing "crc version" succeeds
+        Then stderr should be empty
+        And stdout should contain "version:"
+
+    @darwin @linux @windows
+    Scenario: CRC help
+        When executing "crc --help" succeeds
+        Then stdout should contain "Usage:"
+        And stdout should contain "Available Commands:"
```

Comment:
we might need to check for a subset of commands that should NEVER be hidden/always available.

or perhaps even exclude certain things (since with Minishift we accidently added test commands).

Reply:
I added checks for `help` and `version`, as well as `setup`, `start`, `stop`, `delete`, and `status`. Seemed like the "core" ones.